### PR TITLE
theming.md: automate _root.scss.

### LINKED ISSUE #29324: Automate _root:scss in theming page
It'll probably need some string handling and reading the generated CSS with Hugo. Not sure if it's worth it, but we should look into it.

Refs #29323

### PR DESCRIPTION
**Pull request by XhmikosR**
TODO: fix font variables.

Any ideas welcome.

Fixes #29324 

Preview: <https://deploy-preview-29326--twbs-bootstrap.netlify.com/docs/4.3/getting-started/theming/#available-variables>

### CODE REVIEW

File: site/content/docs/4.3/getting-started/theming.md [hunk 1]
```diff
@@ -370,15 +370,19 @@ Here are the variables we include (note that the `:root` is required). They're l
 
 {{< highlight css >}}
 {{< root.inline >}}
+{{- $variables := readFile "scss/_variables.scss" -}}
+{{- $fontSansSerif := findRE "(\\$font-family-sans-serif:\\s+)(.*)(!default)" $variables -}}
```
Comment (XhmikosR) [inline site/content/docs/4.3/getting-started/theming.md]: @bep: do you have any suggestions how one can get a capturing group from `findRE`?

**Comment by XhmikosR** [comment]
OK, this works.

Not sure about the performance penalty, but I can't find another way to automate this...

**Comment by MartijnCuppens** [comment]
> Not sure about the performance penalty

`:root` is defined quite on top of the file, so it should be fine I guess

**Review by MartijnCuppens — APPROVED** [review]

**Comment by XhmikosR** [comment]
>`:root` is defined quite on top of the file, so it should be fine I guess

We still have to read the whole file and parse it.

**Comment by MartijnCuppens** [comment]
> We still have to read the whole file and parse it.

Yup, I know, worth it imo

**Pull request merged**